source/lac/scalapack.cc

```diff
@@ -1059,7 +1059,7 @@ ScaLAPACKMatrix<NumberType>::eigenpairs_symmetric_MRRR(const bool compute_eigenv
 
   // Number of eigenvectors to be returned;
   // Upon successful exit the first m=nz columns contain the selected eigenvectors (only if jobz=='V').
-  int nz=0;
```

Comment:
Is this needed?

Reply:
This looks like an independent change. Keep this for a separate pull request if you think this is necessary.